packages/attribute-registry-readmodel-writer/kubernetes/dev/configmap.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: interop-be-attribute-registry-readmodel-writer
+  namespace: dev-refactor
+data:
+  KAFKA_CLIENT_ID: "dev-pagopa-interop-attribute-registry"
+  KAFKA_GROUP_ID: "attribute-registry.public.event"
```

```suggestion
  KAFKA_GROUP_ID: "attribute-readmodel-writer"
```
@galales do you prefer the verbose version `attribute-registry-readmodel-writer` instead?

---

the shorter version is ok
(spoiler: at some point `attribute-registry` will be renamed to `attribute`)